### PR TITLE
Added widget tests

### LINKED ISSUE #2: Add screen to show list of tasks
We need to create a second screen to show a list of all of the tasks that have been tracked. 

To start, you can add a new folder called `screens` in the `lib` directory. Inside the newly created `lib/screens` you can add `task_list.dart` where we will put our code. In `lib/main.dart`, you can then import the `task_list.dart` file add an action in the app bar to navigate from the main page to our new task list page (see [general Flutter navigation docs](https://flutter.dev/docs/cookbook/navigation/navigation-basics) and [a more specific example of adding a navigation button to the app bar](https://codelabs.developers.google.com/codelabs/first-flutter-app-pt2#5)).

The task list screen should display a [list](https://flutter.dev/docs/cookbook/lists/mixed-list) of hard-coded tasks (at least 5). Each task needs to have a description  (eg. "Do physics homework"), a category (eg. "Education"), and an amount of time (eg. 50 minutes). Issue #3 deals with creating a `Task` class so for now you can just use any representation you want for the array of tasks (can by hacky for now). You can choose how to display each of these attributes in the list item.

Please also write [widget tests](https://flutter.dev/docs/testing#widget-tests) for the screen in the `test` folder.

### PR DESCRIPTION
**Pull request by ShaunDonn**
Icon list button to navigate from main screen to task list screen
Tests of icon list button (including navigation) and presence of tasks on task list page

### CODE REVIEW

File: lib/src/screens/add_task.dart [hunk 1]
```diff
@@ -1,9 +1,8 @@
 import 'package:flutter/material.dart';
 
-// TODO: Replace categories to use Task class after Issue #3 is completed
-// TODO: Refactor styles to use global theming after more designer guidance
```
Comment (valarao) [inline lib/src/screens/add_task.dart:4]: I wouldn't delete these TODOs. Good to have so we don't forget about the changes we need to make to this file.

File: test/widgets/top_navigation_test.dart [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+void main() {
+  testWidgets('Top Navigation test', (WidgetTester tester) async {
+    await tester.pumpWidget(MaterialApp(
+        title: 'dummy_app',
+        home: Scaffold(
+          appBar: TopNavigation(),
+          body: Container(),
+        )));
+    expect(find.byKey(Key('task_list_icon')), findsOneWidget);
```
Comment (valarao) [inline test/widgets/top_navigation_test.dart:16]: It's nice to have comments for front-end tests. I recommend something like:
`// Click task list icon and assert page navigates to Task List Screen`

File: lib/src/screens/task_list.dart [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    return Scaffold(
+      key: Key('task_list_screen'),
+      appBar: AppBar(title: Text('Task List')),
+      body: BodyLayout(),
+    );
+  }
+}
+
+class BodyLayout extends StatelessWidget {
+  List<String> titles = [
```
Comment (valarao) [inline lib/src/screens/task_list.dart:16]: Maybe add TODO comments above the `title` and `subtitles` lists here like: 
`TODO: Replace after creating a Task class`

**Review by valarao — COMMENTED** [review]
Overall, looks really good! :)

I just left a few minor comments about in-file documentation (IMO they're not imperative to merging in so feel free to ignore them). For future front-end focused PRs, you might want to add a demo `gif`/`png` so reviewers can see the changes made.

Great work @ShaunDonn!

File: lib/src/screens/add_task.dart (hunk 1 above)
Comment (sarahdagger) [reply lib/src/screens/add_task.dart:4]: Just updated this - I imagine it was a merge conflict issue

File: lib/src/screens/task_list.dart [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    'Apartment - 74 minutes',
+    'Assignments - 23 minutes',
+    'Assignments - 201 minutes'
+  ];
+  @override
+  Widget build(BuildContext context) {
+    return _myListView(context);
+  }
+
+  Widget _myListView(BuildContext context) {
```
Comment (sarahdagger) [inline lib/src/screens/task_list.dart]: Bit of a nitpick, but a more descriptive name than `_myListView` would make it a bit easier/faster to read and understand the code.

**Review by valarao — APPROVED** [review]
Thanks for resolving the comments!

**Pull request merged**